lightning/src/ln/channel.rs

```diff
@@ -3665,6 +3664,12 @@ impl<Signer: Sign> Channel<Signer> {
 			},
 		};
 
+		for outp in closing_tx.output.iter() {
+			if !outp.script_pubkey.is_witness_program() && outp.value < MAX_STD_OUTPUT_DUST_LIMIT_SATOSHIS {
```

Comment:
Wait, the closing requirement implemented by this change is "Require cooperative close payout scripts to be Segwit" ? If so why also checking `MAX_STD_OUTPUT_DUST_LIMIT_SATOSHIS`, we want to reject non-segwit output no matter their output values ?

Reply:
The new requirement is "our users have to use segwit scripts, but the counterparty can use something else, but if they do we may have to force close". I tweaked the commit message text to be more clear.